docs/build/multi-chain.md

```diff
@@ -0,0 +1,42 @@
+# Multi-Chain Support
+
+- We support indexing multiple networks into to the same database schema. 
+
+- This is useful if you want to gather data from multiple network together in one place. For example capturing transaction data from seperate network (like kusama and polkadot) or capturing xcm data.
+
+## Notes about this feature:
+- It will create multiple metadata tables which you can query using the chainId of the particular subquery project [see below](/build/multi-chain.html#querying)
+- You need to run multiple subql/node instances for each project you are indexing
+- The flag --multi-chain must be enabled from the start of indexing and for all projects. 
```

Comment:
If they forgot to do this, what do they need to do to enable it in the future? Clear DB?

Reply:
Ah yes clear db, I'll update.